tests/test_lambdas/test_broker.py

```diff
@@ -0,0 +1,94 @@
+from lbz.exceptions import NotFound
+from lbz.lambdas import LambdaBroker, LambdaResponse, LambdaResult, lambda_ok_response
+
+
+def simple_func(*_event: dict) -> LambdaResponse:
+    return lambda_ok_response()
+
+
+class TestEventBroker:
+    def test_broker_works_properly_when_data_is_provided(self) -> None:
+        def func(_event: dict) -> LambdaResponse:
```

Comment:
Also, this should not be named `_event` as handlers accept only `data` (part of the event - payload).